### PR TITLE
Allow "sequential proxy" work with POST, PUT and DELETE methods

### PR DESCRIPTION
**Pull request by Alphyron**
You can use now the sequential proxy with POST, PUT and DELETE methods if you only use GET methods before.
#359

### CODE REVIEW

File: router/chi/router.go [hunk 1]
```diff
@@ -116,15 +116,32 @@ func (r chiRouter) registerKrakendEndpoints(endpoints []*config.EndpointConfig)
...
 }
 
-func (r chiRouter) registerKrakendEndpoint(method, path string, handler http.HandlerFunc, totBackends int) {
+func (r chiRouter) registerKrakendEndpoint(method string, endpoint *config.EndpointConfig, handler http.HandlerFunc, totBackends int) {
 	method = strings.ToTitle(method)
+	path := endpoint.Endpoint
 	if method != http.MethodGet && totBackends > 1 {
-		r.cfg.Logger.Error(method, "endpoints must have a single backend! Ignoring", path)
-		return
+		if endpoint.ExtraConfig["github.com/devopsfaith/krakend/proxy"] == nil {
```
Comment (kpacha) [inline router/chi/router.go:127]: I'd use the constant `proxy.Namespace`

Comment (Alphyron) [reply router/chi/router.go:127]: Ah, okay. Thanks, I didn't know that this constant exists.

**Comment by Alphyron** [comment]
Yes I can extract them in a own function. I will do it later.
I try to create some tests, but I am not good at it.

**Comment by kpacha** [comment]
integration tests are here: https://github.com/devopsfaith/krakend/tree/master/test

let me know if I can help you in any way

**Comment by Alphyron** [comment]
Ah okay, thanks for it

**Comment by kpacha** [comment]
amazing! thanks for the contribution!!!

**Comment by Alphyron** [comment]
You're welcome. Thanks for this project :D

**Pull request merged**